app/src/main/java/org/mozilla/fenix/settings/logins/SavedLoginsView.kt

```diff
@@ -13,7 +13,9 @@ import android.widget.FrameLayout
 import androidx.core.view.isVisible
 import androidx.recyclerview.widget.LinearLayoutManager
 import kotlinx.android.extensions.LayoutContainer
+import kotlinx.android.synthetic.main.component_history.view.*
```

Comment:
I don't think we want this import from history?

Reply:
Not sure how this escaped ktlint 😯 . Thanks for the review! 😄